pkgs/development/python-modules/python-bip32/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pytestCheckHook
+, pythonOlder
+, base58
+, coincurve
+}:
+
+buildPythonPackage rec {
+  pname = "python-bip32";
+  version = "3.4";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    inherit version;
```

Comment:
```suggestion
```
I think we don't need this.

Reply:
Sorry yeah, that was left over from attempting to use `fetchPypi` ><